assets/csidriveroperators/shared-resource/09_daemonset.yaml

```diff
@@ -0,0 +1,59 @@
+apiVersion: apps/v1
+kind: Daemonset
```

Comment:
We usually don't deploy *operators* as DaemonSets. Please use a Deployment and copy some recent one - we added support for SNO (that should run 1 replica only) and for hypershift (which does not have master nodes).